wee_slack.py

```diff
@@ -3877,6 +3939,9 @@ def initiate_connection(token, retries=3):
     if w.register(SCRIPT_NAME, SCRIPT_AUTHOR, SCRIPT_VERSION, SCRIPT_LICENSE,
                   SCRIPT_DESC, "script_unloaded", ""):
 
+        global proxy
+        proxy = ProxyWrapper()
```

Comment:
It's probably better to just call this the places you use it so you pick up any changes in weechat's settings.